include/spdlog/details/os.h

```diff
@@ -135,8 +195,17 @@ inline int fopen_s(FILE** fp, const std::string& filename, const char* mode)
     *fp = _fsopen((filename.c_str()), mode, _SH_DENYWR);
     return *fp == nullptr;
 #else
+    std::list<std::string> dirs;
```

Comment:
i am not sure its good idea to check dirs on every file open. should be done only once at some initial stage.